packages/compass-e2e-tests/tests/collection-import.test.ts

```diff
@@ -19,12 +19,14 @@ async function importJSONFile(browser: CompassBrowser, jsonPath: string) {
   await insertDocumentOption.waitForDisplayed();
   await browser.clickVisible(Selectors.ImportFileOption);
 
-  // wait for the modal to appear and select the file
+  // Select the file.
+  await browser.selectFile(Selectors.ImportFileInput, jsonPath);
```

Comment:
Something I thought about with all these selectFile bits moved around: I wonder if we should have something somewhere changing the file (ie. maybe clicking again and then cancel, then again and re-selecting it) to make sure all that works. We can also add it later or I can add it to the new tests for the new backend.

Reply:
I'm also a little surprised that the [selectFile helper](https://github.com/mongodb-js/compass/blob/6b97a8ff0f178c31cf6c873948a76865b1958005/packages/compass-e2e-tests/helpers/commands/select-file.ts#L1-L31) works with the file dialog already open and the modal still closed. But I'm not gonna think about it too much if it works :)

Reply:
Sounds good, let's add a test for that. Didn't include it in this pr, will in the file input ui updates if not sooner.